code/__DEFINES/callbacks.dm

```diff
@@ -1,3 +1,4 @@
 #define GLOBAL_PROC	"some_magic_bullshit"
 
-#define CALLBACK new /datum/callback
\ No newline at end of file
+#define CALLBACK new /datum/callback
+#define INVOKE ;new /datum/callback/immediate	//semicolon to prevent shit from trying to assign us/use us as an arg
```

Comment:
Oh wait, we're just bypassing addtimer completely, since addtimer just calls this async stuff. Gotcha.